Runners/ConsoleApp/Program.cs

```diff
@@ -15,12 +16,11 @@ class Program
     {
         static void Main(string[] args)
         {
-            var pluginExecutor = new PluginExecutor(new MemoryDal(), new PluginsMenu(), new PluginsManager());
+            var pluginExecutor = new PluginExecutor(new DbDal(), new PluginsMenu(), new PluginsManager());
 
```

Comment:
This file is not formatted, Please use the format document option here (CTRL K, D)
You can set vs 2022 to auto format on save, look [here](https://stackoverflow.com/questions/70460978/format-code-and-sort-using-on-save-in-visual-studio-2022)